rasp/crawlers/base.py

```diff
@@ -0,0 +1,19 @@
+class BaseBoundedCrawler(object):
+    def run(self):
+        raise NotImplemented("run not implemented for {}"
+                             .format(str(self.__class__.__name__)))
+
+
+class DefaultBoundedCrawler(BaseBoundedCrawler):
```

Comment:
Unbounded in my mind would sit on an external queue, which some thing was putting messages into and the crawler would be able to submit new urls to that queue as well.  So a bit more involved.  I'll pull in develop and make other changes when I get a chance, thanks for the notes.